### PR TITLE
Issue #418

### PR DESCRIPTION
**Pull request by kirjaamo**
Added examples of session-based method changes. Fixed one rst problem, which caused one option to appear in an incorrect place in the table.

### CODE REVIEW

File: galeracluster/source/documentation/mysql-wsrep-options.rst [hunk 1]
```diff
@@ -1260,7 +1260,7 @@ Defines the Online Schema Upgrade method the node uses to replicate :abbr:`DDL (
    "Dynamic Variable", "Yes"
    "Permitted Values", "Enumeration"
    "Default Value", "``TOI``"
-   "Valid Values", "``TOI``, ``RSU``", ``NBO``"
+   "Valid Values", "``TOI``, ``RSU``, ``NBO``"
    "Initial Version", "MySQL-wsrep: 5.5.17-22.3, MariaDB: 5.5.21"
    "Initial Version, NBO", "MariaDB Enterprise Server Version 10.5, Percona XtraDB Cluster 8.0.25-15.1"
```
Comment (byte) [inline galeracluster/source/documentation/mysql-wsrep-options.rst:1265]: We have made no mention of NBO in Galera Cluster Enterprise Edition (EE)? We must

File: galeracluster/source/documentation/schema-upgrades.rst [hunk 2]
```diff
@@ -93,12 +93,19 @@ You can set the method for online schema changes by using the ``wsrep_OSU_method
...
    pair: Descriptions; Total Order Isolation
 
-When you want an online schema change to replicate through the cluster and don't care that other transactions will be blocked while the cluster processes the :abbr:`DDL (Data Definition Language)` statements, use the :term:`Total Order Isolation` method. You would do this with the ``SET`` statement like so:
+When you want an online schema change to replicate through the cluster and don't care that other transactions will be blocked while the cluster processes the :abbr:`DDL (Data Definition Language)` statements, use the :term:`Total Order Isolation` method. You can do this with a global ``SET`` statement, as follows:
 
 .. code-block:: mysql
 
    SET GLOBAL wsrep_OSU_method='TOI';
 
+And with a session-based ``SET`` statement, as follows:
```
Comment (byte) [inline galeracluster/source/documentation/schema-upgrades.rst:102]: can we somehow make this a bit more verbose

**Review by byte — CHANGES_REQUESTED** [review]

File: galeracluster/source/documentation/mysql-wsrep-options.rst (hunk 1 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/mysql-wsrep-options.rst:1265]: How do we do this?

**Review by byte — APPROVED** [review]
push

**Pull request merged**